lib/utils.js

```diff
@@ -54,6 +54,13 @@ function setIfValue(target, path, value) {
   set(target, path, value)
 }
 
+function uniqueOnly(a) {
```

Comment:
@moranthomas I am a little bit confused by the method/param naming.
I do understand that this method is supposed to deduplicate the data somehow based on properties, but I am under the impression we should use a more eloquent naming and adhere to general JS best practices: "**A function name should be a verb or a verb phrase, and it needs to communicate its intent, as well as the order and intent of the arguments**."

Would like also @storrisi to share his comments.

Other than that, I am happy with the implementation.

Reply:
Agreed, the function and it's params could be more clear in their intent by naming convention. Will implement.